tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -2,23 +2,28 @@
 Library             SeleniumLibrary
 Resource            ../../../Resources/Page/OCPLogin/OCPLogin.resource
 Resource            ../../../Resources/OCP.resource
+Resource            ../../../Resources/Page/ODH/AiApps/AiApps.resource
 
 Test Setup          Dashboard Test Setup
 Test Teardown       Dashboard Test Teardown
 
 
 *** Variables ***
 ${pachyderm_container_name}     Pachyderm
+${pachyderm_operator_name}      Pachyderm
 ${pachyderm_appname}            pachyderm
 
 
+
 *** Test Cases ***
 Verify Pachyderm Can Be Installed Using OpenShift Console
-    [Tags]    Tier2    ODS-1137
+    [Tags]      Tier2
+    ...         ODS-1137    ODS-1138
     Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
+    Create Tabname Instance For Installed Operator        ${pachyderm_container_name}   ${pachyderm_container_name}     redhat-ods-applications
```

Comment:
I think there is a missing step which is reported in ODS-1138, that is the secret creation